tokens/utils.js

```diff
@@ -162,7 +162,37 @@ async function transformInPath(location, variablesMap, transformType = 'definiti
   }
 }
 
+
+function createIndexFile(buildDir, isTheme, themeVariant){
```

Comment:
nit: It may be worth changing the function signature here to be more like kwargs in python, e.g. `function createIndexFile({ buildDir, isTheme, themeVariant }) {`. Makes it more explicit what args you're passing in at first glance.